probe-rs/src/probe/jlink/mod.rs

```diff
@@ -637,6 +639,11 @@ impl DebugProbe for JLink {
             Err((self, DebugProbeError::InterfaceNotAvailable("SWD/ARM")))
         }
     }
+
+    // self.handle.read_target_voltage() returns the voltage as an integer in mV.
```

Comment:
Here, `self.handle.read_target_voltage()` refers to the inner function’s return value, which I divide by 1000 and cast as an f32 to convert to volts we can return from the trait function. When I code maths with magic numbers like this, I like to explain what it’s doing, even though it may be obvious from context. I’m happy to clarify or remove the comment if it’s confusing or unnecessary though.

Reply:
Ah my bad then ofc! Yould you move the comment above the actual statement then? I was under the impression this documents the function; in hindsight I feel silly lol.